diff --git a/code.java b/code.java
new file mode 100644
index 0000000..92b164a
--- /dev/null
+++ b/code.java
@@ -0,0 +1,7 @@
+public class static void main (String[] argv) {
+    int a = 5;
+    int b = 13;
+    int c = a + b
+
+    System.out.println(a + " + " + b + " = " + c);
+}
\ No newline at end of file
diff --git a/newFile b/newFile
new file mode 100644
index 0000000..2920714
--- /dev/null
+++ b/newFile
@@ -0,0 +1 @@
+This is new file for admin repository. Maked it for testing our site...
\ No newline at end of file
